docs/feature_macros.md

```diff
@@ -195,6 +195,49 @@ This will clear all mods currently pressed.
 
 This will clear all keys besides the mods currently pressed.
 
+### Advanced Example: 
+
+#### Super ALT/TAB
```

Comment:
```suggestion
### Super ALT/TAB
```

Reply:
```suggestion
### Super Alt+Tab
```
`Alt+Tab` makes more sense imo; `ALT/TAB` makes it sound like it's either one or the other.

Reply:
It's a little of both? :D

Reply:
Alt↯Tab :smile: